core/domain/classifier_services.py

```diff
@@ -333,7 +335,7 @@ def _update_scheduled_check_time_for_new_training_job(job_id):
     """
     classifier_training_job_model = (
         classifier_models.ClassifierTrainingJobModel.get(job_id))
-
+    
```

Comment:
Extra spaces on this line, please remove.

Reply:
Done